python/orca/src/bigdl/orca/ray/model.py

```diff
@@ -0,0 +1,218 @@
+# This file is adapted from https://github.com/ray-project/ray/blob
```

Comment:
this should be an example, not part of the main library

Reply:
Yeah, but the original way to import model in async_parameter_server.py is not working in yarn-test and k8s-test, put model.py in src could solve problems we met before.

Reply:
"put model.py in src" is not the correct solution; what if the users need to build their own code on yarn or k8s? They cannot put their code in src.

Reply:
yes, I'll think about other solutions.